tool/tctl/common/acl_command.go

```diff
@@ -190,33 +191,32 @@ func (c *ACLCommand) UsersRemove(ctx context.Context, client *authclient.Client)
 
 // UsersList will list the users in an access list.
 func (c *ACLCommand) UsersList(ctx context.Context, client *authclient.Client) error {
-	members, nextToken, err := client.AccessListClient().ListAccessListMembers(ctx, c.accessListName, 0, "")
-	if err != nil {
-		return trace.Wrap(err)
-	}
-
-	if len(members) == 0 {
-		fmt.Printf("No members found for access list %s.\nYou may not have access to see the members for this list.\n", c.accessListName)
-		return nil
-	}
-
-	fmt.Printf("Members of %s:\n", c.accessListName)
+	var allMembers []*accesslist.AccessListMember
+	var nextToken string
 	for {
-		for _, member := range members {
-			fmt.Printf("- %s\n", member.Spec.Name)
+		members, nextToken, err := client.AccessListClient().ListAccessListMembers(ctx, c.accessListName, 0, nextToken)
+		if err != nil {
+			return trace.Wrap(err)
 		}
-
+		allMembers = append(allMembers, members...)
 		if nextToken == "" {
 			break
 		}
-
-		members, nextToken, err = client.AccessListClient().ListAccessListMembers(ctx, c.accessListName, 0, nextToken)
-		if err != nil {
-			return trace.Wrap(err)
+	}
+	switch c.format {
+	case teleport.JSON:
+		return trace.Wrap(utils.WriteJSONArray(os.Stdout, allMembers))
+	default:
```

Comment:
I probably would add a case for `teleport.Text` and then made the default case fail with an 'unsupported format' error, just to be safe.